mne/decoding/transformer.py

```diff
@@ -147,13 +147,13 @@ def inverse_transform(self, epochs_data, y=None):
         return X
 
 
+@deprecated("Class 'ConcatenateChannels' has been renamed to "
+            "'EpochsVectorizer' and will be removed in release 0.11.")
 class ConcatenateChannels(TransformerMixin):
     """Concatenates data from different channels into a single feature vector
 
     Parameters
     ----------
-    info : instance of Info
-        The measurement info.
```

Comment:
Don't fix it this way, fix it by adding the function or class name here:

https://github.com/mne-tools/mne-python/blob/master/mne/tests/test_docstring_parameters.py#L64